### PR TITLE
[API compatibility] paddle.nn.functional.layer_norm to support parameter alias

### PR DESCRIPTION
**Pull request by hushenwei2000**
### PR Category
Operator Mechanism

### PR Types
New features

### Description
layer_norm 支持 Torch 的传参方法。

Paddle 原 API 签名：
`paddle.nn.functional.layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None)`

Torch API 签名：
`torch.nn.functional.layer_norm(input, normalized_shape, weight=None, bias=None, eps=1e-05)`

此 PR 添加以下特性：

1. `x` 支持别名 `input`
1. `epsilon` 支持别名 `eps`

pcard-67164

### CODE REVIEW

**Review by DanielSun11 — APPROVED** [review]
LGTM
